### PR TITLE
Add tlcv2.0_oa

### PR DESCRIPTION
**Pull request by wannaphong**
`tlcv2.0_oa` is a dataset that made for Open Assistant Text-only format. It was build from Thai Literature Corpora (TLC).

Thai Literature Corpora (TLC) is Corpora of machine-ingestible Thai classical literature texts like gutenberg. [https://attapol.github.io/tlc.html](https://attapol.github.io/tlc.html)

Hugging Face Datasets: [pythainlp/tlcv2.0_oa](https://huggingface.co/datasets/pythainlp/tlcv2.0_oa)

### CODE REVIEW

**Comment by bitplane** [comment]
Can you strip the outputs from the notebook please? It's a bit large! Also rebase + squash commits if possible.

File: data/datasets/tlcv2.0_oa/loading_script.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+from datasets import load_dataset
```
Comment (bitplane) [inline data/datasets/tlcv2.0_oa/loading_script.py:1]: We don't need this file anymore, it's automatically done via `__init__.py` in the parent dir 🙂

**Review by bitplane — CHANGES_REQUESTED** [review]
Looks good, just need to remove the loading file like in the other datasets :)

**Review by Vechtomov — APPROVED** [review]

**Comment by bitplane** [comment]
Rebased on `main`, fixed merge conflicts and kept contributor commits rather than squash :)

**Review by bitplane — APPROVED** [review]

**Comment by olliestanley** [comment]
@bitplane I think in general we decided to squash all PRs, to keep history clean. If you use "Squash and merge" in GitHub it will list the author of the commit as the person who made the PR, and anyone else who committed will also receive credit as a co-author :)

**Comment by bitplane** [comment]
Sorry, I merged this before I saw the discussion on Discord. I'll squash in future :)

**Pull request merged**